src/components/AddressInput/AddressInput.st.css

```diff
@@ -0,0 +1,100 @@
+:import {
+    -st-from: "wix-ui-core/dist/src/components/AddressInput/AddressInput.st.css";
+    -st-default: AddressInput;
+}
+
+:import {
+    -st-from: "../../palette.st.css";
+    -st-named: primaryText, mainHover, main, mainMuted, mainMutedHover, disabledText, inputHover, primaryLightText, inputBorderColor, inputDisabledBorderColor, inputDisabledBorderColor, inputDisabledHoverBorderColor, danger, backgroundSecondary;
+}
+
+:import {
+    -st-from: "../Input/Input.st.css";
+    -st-default: Input;
+}
+
+:import {
+    -st-from: "../Input/InputStates.st.css";
+    -st-named: input-large, input-medium, input-small;
+}
+
+:import {
+    -st-from: "../../typography.st.css";
+    -st-named: fontRoman;
+}
+
+.root {
+    -st-extends: AddressInput;
+    -st-states: size(enum(large, medium, small));
+    font-family: value(fontRoman);
+    width: 100%;
+}
+
+.root > [data-hook="popover-content"] {
+    border: none;
+    border-radius: 8px;
+    box-shadow: 0 0 16px 0 rgba(0, 0, 0, 0.14);
```

Comment:
We have a new organized box-shadows that Ben specified.
See `shadows.st.css` file please.